packages/angular/cli/tasks/install-package.ts

```diff
@@ -143,7 +143,7 @@ export function runTempPackageBin(
     throw new Error(`Cannot locate bin for temporary package: ${packageNameNoVersion}.`);
   }
 
-  const argv = [binPath, ...args];
+  const argv = ['binPath', ...args];
```

Comment:
```suggestion
  const argv = [`'${binPath}'`, ...args];
```

Reply:
Could you review the changes made, thank you.